src/components/Date/SimpleDate.jsx

```diff
@@ -0,0 +1,211 @@
+import React, { useRef } from 'react';
+import PropTypes from 'prop-types';
+import _ from 'lodash';
+
+import Select from '../Select/Select';
+import NumberInput from '../NumberInput/NumberInput';
+import { months, days } from '../../helpers/options-for-select.js';
+
+const handleChange = (path, newValue, oldDate, onValueChange) => {
+  if (!onValueChange) return;
+
+  const date = {
+    ...oldDate,
+    [path]: newValue,
+  };
+
+  if (!date.month.value) {
+    date.day.value = '';
+  }
+
+  onValueChange(date);
+};
+
+// TODO: Refactor the date prop to be either a string or an object of strings
+
+/**
+ * A simple date picker with only display logic. For built-in validation, use the Date component.
+ */
+export const SimpleDate = ({
+  errorMessage,
+  date,
+  label,
+  name,
+  required,
+  ariaDescribedby,
+  minYear,
+  maxYear,
+  onValueChange,
+  onBlur,
+}) => {
+  const inputId = useRef(_.uniqueId('date-input-'));
+  const { day, month, year } = date;
+
+  let daysForSelectedMonth = [];
+  if (month.value) {
+    daysForSelectedMonth = days[month.value];
+  }
```

Comment:
I see that this is copy-pasted from the old `Date`, but would this work? I like that it would let us use a `const` so a reader knows that the value won't change.

```js
const daysForSelectedMonth = days?[month.value] || [];
```

Reply:
I'm not sure if we have optional chaining enabled in our Babel setup in this repo, but I can give it a shot! :+1: This looks cleaner to me.

But for what it's worth, as far as arrays (and Objects and Maps and honestly anything other than primitives) go, we still have no assurance that the value won't change. We just know the variable won't be reassigned. `const` != `immutable`.

Reply:
I went with this to preserve the same functionality if `month.value` exists but is not a key of `days`:
```js
  const daysForSelectedMonth = month.value ? days[month.value] : [];
```

Reply:
Yeah, good point about non-primitives. At least it indicates that it won't be reassigned.